ci/rubyonrails.yml

```diff
@@ -0,0 +1,66 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This workflow will install a prebuilt Ruby version, install dependencies, and
+# run tests and linters.
+name: "Ruby on Rails CI"
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      DB: rails_test
+      USER: rails
+      PASSWORD: password
+      PORT: 5432
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - "$${{env.PORT}}:$${{env.PORT}}"
+        env:
+          POSTGRES_DB: ${{env.DB}}
+          POSTGRES_USER: ${{env.USER}}
+          POSTGRES_PASSWORD: ${{env.PASSWORD}}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      # Add or replace dependency steps here
+      - name: Install Ruby and gems
+        uses: ruby/setup-ruby@v1
```

Comment:
Please use the full SHA for any action which is not directly published by GitHub. See "general notes" section [here](https://github.com/actions/starter-workflows/blob/main/.github/pull_request_template.md).

Reply:
Done.